src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Android/StorePal.cs

```diff
@@ -46,5 +55,25 @@ public static IStorePal FromSystemStore(string storeName, StoreLocation storeLoc
         {
             throw new NotImplementedException(nameof(FromSystemStore));
         }
+
+        private static ICertificatePal[] ReadPkcs12Collection(ReadOnlySpan<byte> rawData, SafePasswordHandle password)
+        {
+            using var reader = new AndroidPkcs12Reader(rawData);
+            reader.Decrypt(password);
+
+            ICertificatePal[] certs = new ICertificatePal[reader.GetCertCount()];
+            int idx = 0;
+            foreach (UnixPkcs12Reader.CertAndKey certAndKey in reader.EnumerateAll())
+            {
+                AndroidCertificatePal pal = (AndroidCertificatePal)certAndKey.Cert!;
+                if (certAndKey.Key != null)
+                    pal.SetPrivateKey(AndroidPkcs12Reader.GetPrivateKey(certAndKey.Key));
```

Comment:
Personally, I'd like every if that isn't a method preamble of if-null-throw (or similar if-something-throw) to be braced.  But this one should definitely be braced by the style rules since it's within the foreach ("the block ... and must not be nested in other statement blocks that use braces").